packages/opam-monorepo/opam-monorepo.0.2.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis: "Assemble and manage fully vendored Dune repositories"
+description: """
+The opam monorepo plugin provides a convenient interface to bridge the
+opam package manager with having a local copy of all the source
+code required to build a project using the dune build tool."""
+maintainer: ["anil@recoil.org"]
+authors: [
+  "Anil Madhavapeddy" "Nathan Rebours" "Lucas Pluvinage" "Jules Aguillon"
+]
+license: "ISC"
+homepage: "https://github.com/ocamllabs/opam-monorepo"
+bug-reports: "https://github.com/ocamllabs/opam-monorepo/issues"
+depends: [
+  "dune" {>= "2.6"}
+  "ocaml" {>= "4.08.0" & < "4.12"}
```

Comment:
Is there a reason why this is unavailable on OCaml 4.12?

Reply:
Yeah one of the vendored deps doesn't build on 4.12 yet I think! If that's fine by you, I'll take another look at 4.12 compat when the beta is released!

Reply:
Ah I see, base and stdlib-shims, right. Thanks!